.env.example

```diff
@@ -1,17 +1,21 @@
-DB_USERNAME=
-DB_PASSWORD=
-DB_DATABASE=
-DB_HOST=
+NODE_ENV=development
```

Comment:
Hello @codexempire , Could you please remove the information in the .env.example here? these information should not be here